rust/parquet/src/arrow/arrow_reader.rs

```diff
@@ -304,6 +304,45 @@ mod tests {
         >(2, 100, 2, message_type, 15, 50, converter);
     }
 
+    #[test]
```

Comment:
Both these tests fail without the changes in this PR.

I don't like the copy/paste nature of these tests and I plan a minor PR building on this one proposing how to remove the duplication and make the tests easier to read.

Reply:
https://github.com/apache/arrow/pull/8009